pyproject.toml

```diff
@@ -20,7 +20,10 @@ dependencies = [
   'astpretty ~= 3.0',
   'cuquantum-cu11 ~= 24.03',
   'graphlib-backport >= 1.0',
-  'numpy >= 1.24'
+  'numpy >= 1.24',
+  'nvidia-cublas-cu11 == 11.11.3.6; platform_machine == "x86_64"',
+  'nvidia-cuda-runtime-cu11 == 11.8.89; platform_machine == "x86_64"',
+  'nvidia-cusolver-cu11 == 11.4.1.48; platform_machine == "x86_64"'
```

Comment:
I know not technically part of this PR, but while we are at it:
Since we needed to update the devdeps image to accommodate h5py, I think the optional dependencies for chemistry may need to be updated as well.